domain/src/main/java/template/android/com/domain/usecase/conference/DoesConferenceExistUseCaseImpl.kt

```diff
@@ -0,0 +1,11 @@
+package template.android.com.domain.usecase.conference
+
+import io.reactivex.Observable
+import template.android.com.domain.repository.ConferenceRepository
+
+class DoesConferenceExistUseCaseImpl (private val conferenceRepository: ConferenceRepository) : DoesConferenceExistUseCase {
+    override fun execute(id: String): Observable<Boolean> {
+        // TODO: connect with firebase. Where to implement firebase? Should it be through repository and then implementation in data/firebase package?
```

Comment:
Da, dobro si to zamislio. Ako će se stvari komplicirati može se kasnijeto još doraditi, al ovo je sasvim dobro!